src/testutils.ts

```diff
@@ -24,6 +24,8 @@ export interface TypeTestCase {
     invalidConversions?: [input: unknown, message: string | string[] | RegExp][];
 }
 
+export const stripped = Symbol('stripped');
```

Comment:
Seen as this is a very important property and that the meaning of the name could be interpreted differently, this could use a bit of jsDoc.